Automattic/newspack-theme
# Add editor styles for style packs

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds editor styles for Style Pack 1.

Changes include adding a build for the style pack editor styles, and splitting the original styles out into a 'base' (styles that are built into every editor style sheet) and a default scss file (which mixes the default variables with the editor style base). 

It also adds the body background style to the editor styles, so it can be changed.

Closes #47.

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Navigate to Customizer > Style Pack, and switch to Style 1.
3. Compare the front-end to the editor -- confirm that the fonts match (all serif), and that the background is a light grey (the two main differences between Style 1 and the default at the moment). 

### Other information:
